### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by lazzarello**
Update the APT repo to the correct syntax, DRY up some docs, make link clickable

### CODE REVIEW

File: readme-setup.md [hunk 1]
```diff
@@ -28,7 +20,7 @@ or use the debian repository as follows:
 
 ```
 curl https://keybase.io/artfwo/pgp_keys.asc | sudo apt-key add -
-echo "deb https://package.monome.org/ debian/" | sudo tee /etc/apt/sources.list.d/norns.list
+echo "deb https://package.monome.org/ strecth main" | sudo tee /etc/apt/sources.list.d/norns.list
```
Comment (catfact) [inline readme-setup.md]: typo 'strecth' ?

Comment (lazzarello) [reply readme-setup.md]: lol, yeah

Comment (lazzarello) [reply readme-setup.md]: vvv

**Comment by tehn** [comment]
port 5000 is no longer required. everything is on port 80.

**Pull request merged**